applications/FluidDynamicsApplication/custom_utilities/mass_conservation_utility.h

```diff
@@ -55,50 +53,44 @@ namespace Kratos
 /// Utility to modify497 the distances of an embedded object in order to avoid bad intersections
 /// Besides, it also deactivate the full negative distance elements
 
-class KRATOS_API(FLUID_DYNAMICS_APPLICATION) MassConservationCheckProcess : public Process
+class KRATOS_API(FLUID_DYNAMICS_APPLICATION) MassConservationUtility
```

Comment:
I found that keeping it as a process is more flexible (e.g. add the process list to check the mass conservation).